Makefile

```diff
@@ -102,6 +102,9 @@ unit-test: $(TEST_BUILD_DIR) ## Run unit tests
 	@$(GOTOOL) cover -html=$(TEST_BUILD_DIR)/coverage.out -o $(TEST_BUILD_DIR)/coverage.html
 	@printf "\nTotal code coverage: " && $(GOTOOL) cover -func=$(TEST_BUILD_DIR)/coverage.out | grep 'total:' | awk '{print $$3}'
 
+unit-test-with-race-condition-detection: $(TEST_BUILD_DIR) ## Run unit tests with race condition detection
```

Comment:
Can't we use something like `race-unit-test` or `unit-test-with-race`? This feels a little too verbose for my taste.

Reply:
Maybe `race-condition-test` might be better?

Reply:
Yeah `race-condition-test` is definitely the clearest option 👌